tests/test_buku.py

```diff
@@ -114,14 +114,17 @@ def test_get_PoolManager(m_myproxy):
         ),
     ],
 )
-def test_parse_tags(keywords, exp_res):
+@pytest.mark.parametrize('prefix', [None, '', '+', '-'])
```

Comment:
Testing for `edit_input` (`None` means `False`, others mean `True` with the value itself defining what prefix is expected before the list)